### DISCUSSION TITLE
how do i use right click?

### DISCUSSION THREAD

**Discussion by ghost** [gujjwal00/avnc#185] [Q&A]
@gujjwal00 
* I have app in systray
* right clicking [ mouse ] provides option to quit directly
* is there any way to use right click [ mouse ]?

**Comment by gujjwal00** [comment]
Right click should work correctly, unless your device overrides the default right-click behavior and changes it to back-press ( e.g. MIUI does this.)

Which device & Android version are you using? Can you post a screen recording of the issue?
